internal/signalfx-agent/pkg/core/services/rules.go

```diff
@@ -96,7 +97,11 @@ func EvaluateRule(si Endpoint, originalText string, errorOnMissing bool, doValid
 
 func ExecuteRule(originalText string, variables map[string]interface{}) (interface{}, error) {
 	env := utils.MergeInterfaceMaps(ruleFunctions, variables)
-	ruleProg, err := expr.Compile(preprocessRuleText(originalText), expr.Env(env))
+	ruleProg, err := expr.Compile(preprocessRuleText(originalText), expr.Env(env),
+		expr.DisableBuiltin("type"),
```

Comment:
Did a failing test demonstrate this was necessary?

Reply:
No, no test was failing. I don't have a very clear understanding of the usages of `expr.Compile`, so out of concern that it was possible for the expression to make use of a `type` variable, I updated all usages to account for it. Is it unnecessary here?

Reply:
I've removed this change.